### PR TITLE
more doxygen changes

### PR DESCRIPTION
**Pull request by edwardhartnett**
Part of #191 

More doxygen changes.

### CODE REVIEW

**Comment by GeorgeGayno-NOAA** [comment]
@edwardhartnett Please merge the latest updates from 'develop' to your branch.

**Comment by edwardhartnett** [comment]
Done.

File: sorc/orog_mask_tools.fd/orog.fd/mtnlm7_oclsm.f [hunk 1]
```diff
@@ -1,87 +1,78 @@
...
-C     GBYTES     - UNPACK BITS
-C
-C   REMARKS: FORTRAN 9X EXTENSIONS ARE USED.
-C           
-C ATTRIBUTES:
-C   CRAY YMP & IBM AIX 3 5 00C88D5D4C00.
-C
-C$$$
-CFPP$ NOCONCUR F
+C> @file
```
Comment (GeorgeGayno-NOAA) [inline sorc/orog_mask_tools.fd/orog.fd/mtnlm7_oclsm.f:1]: I don't see this file under the file list.

Comment (edwardhartnett) [reply sorc/orog_mask_tools.fd/orog.fd/mtnlm7_oclsm.f:1]: Excellent catch, thank you.

There was a problem in the Doxyfile.in file which I've fixed. I've updated the on-line docs.

Comment (GeorgeGayno-NOAA) [reply sorc/orog_mask_tools.fd/orog.fd/mtnlm7_oclsm.f:1]: I see it now.

**Review by GeorgeGayno-NOAA — APPROVED** [review]
Everything looks ok now.

**Comment by edwardhartnett** [comment]
OK, as discussed, I'll let you do the merging so that you can work this and future PRs into whatever order you need.

Once merged, I'll get started on the next batch of doxygen changes. I have move to a fork, so there is no branch to delete. ;-)

**Pull request merged**